src/cbl-mariner/2.0/crossdeps-builder/Dockerfile

```
@@ -26,6 +26,11 @@ RUN wget https://packages.microsoft.com/cbl-mariner/2.0/prod/extended/x86_64/con
     tdnf install -y \
         libbsd-devel
 
+ENV NODE_VERSION 20.11.1
```

```suggestion
ENV NODE_VERSION=20.11.1
```

the other syntax is deprecated according to docker docs